README.md

```diff
@@ -74,3 +74,52 @@ bash ./configure.sh
 **Note**: If you changed the installation namespace used by the installer you will first need to run `export NAMESPACE=<namespace used>` as the default value is `ai-rhdh`.
 
 For information related to extra configuration options, see [`docs/EXTRA-CONFIG.md`](./docs/EXTRA-CONFIG.md).
+
+## Remote Cluster Configuration
```

Comment:
I vote to keep this info out of the main README since you already have a ncie detailed doc in docs/REMOTE-CLUSTER-CONFIG.md,

maybe under this header you can just say - refer to docs/REMOTE-CLUSTER-CONFIG.md for informatino about how to configure remote cluster